components/appstream-usage-pattern.tsx

```
@@ -0,0 +1,332 @@
+import { Card } from "@/components/ui/card";
+import { Users, Clock, AlertCircle } from "lucide-react";
+import type { AppStreamUsagePattern } from "@/types/appstream";
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "@/components/ui/tooltip";
+
+interface AppStreamUsagePatternProps {
+  value: AppStreamUsagePattern;
+  onChange: (updates: Partial<AppStreamUsagePattern>) => void;
+  maxUsers: number; // Add maxUsers prop to limit concurrent users
+}
+
+export function AppStreamUsagePattern({ value, onChange, maxUsers = 100 }: AppStreamUsagePatternProps) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'AppStreamUsagePattern'. Consider to delete it or rename it.

'AppStreamUsagePattern' is defined here:


(lint/suspicious/noRedeclare)